opnode/contracts/l1block/l1_block_info_raw.go

```diff
@@ -0,0 +1,356 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Can you add a go-generate statement with the abigen command so we can keep the bindings up-to-date? (not here but in the package it is used in, or maybe the `contracts` package in a `doc.go` or something). As well as for the other contract bindings?

Reply:
I can add a script/go-generate to get this working.
My understanding is that the `go:generate` command would go in these files.

Reply:
Ok, I figured out how to do this (in `doc.go`)